tools/reviewer/service/code_review.proto

```diff
@@ -6,29 +6,24 @@ option java_package = "com.google.startupos.tools.reviewer.service";
 option java_outer_classname = "Protos";
 
 import "common/text_difference.proto";
+import "google/protobuf/empty.proto";
 
 service CodeReviewService {
   // Get a text file's contents
   rpc getFile(FileRequest) returns (FileResponse);
-  rpc createDiff(CreateDiffRequest) returns (CreateDiffResponse);
+  rpc createDiff(CreateDiffRequest) returns (.google.protobuf.Empty);
```

Comment:
Why do we need the leading `.` in `.google.protobuf.Empty`?

Reply:
@oferb otherwise it would join it to package (`com.google.startupos.tools.reviewer.service` in our case). It's exactly formulated by `protoc` as:
```
ERROR: /Users/vmax/aa_root_path/head/startup-os/tools/reviewer/service/BUILD:5:1: Generating Descriptor Set proto_library //tools/reviewer/service:code_review_proto failed (Exit 
tools/reviewer/service/code_review.proto:14:46: "google.protobuf.Empty" is resolved to "com.google.protobuf.Empty", which is not defined. The innermost scope is searched first in name resolution. Consider using a leading '.'(i.e., ".google.protobuf.Empty") to start from the outermost scope.
```

Reply:
Got it. Thanks!